pointers-and-errors.md

```diff
@@ -310,7 +310,8 @@ There's some duplication in our tests, lets refactor that out.
 func TestWallet(t *testing.T) {
 
     assertBalance := func(t *testing.T, wallet Wallet, want Bitcoin) {
-        got := wallet.Balance()
+        t.Helper()
+    	got := wallet.Balance()
```

Comment:
@midnightrun Looks like an indent mixed TAB and spaces. Could you keep the same indents with the context?

Reply:
I used `gofmt` for formatting. Didn't change any default here. But looks strange to me too :/

Reply:
Me either. Because gofmt only format go codes but not markdown.

Reply:
Ok, will fix it with another PR. Thanks @pityonline for making me aware :)